### ISSUE TITLE
value 13 in the last line of the soil files

### ISSUE THREAD

**Issue by marianadobre** [rogerlew/wepppy#39]
I forgot to mention this but we need to remove the 13 in the soil files. That is just for the desktop interface. The desktop interface strips that value before putting the file in the runs folder. We have to do the same on the interface. Otherwise, wepp will read it as bedrock depth instead of 10000.

**Comment by rogerlew** [comment]
@marianadobre 

Fixed.
So if the last line was:
`1 13 10000.0 1.08`

It is now:
`1 10000.0 1.08` 

I'll just leave this here...

> Then he decided he wasn’t going to tell anyone that this was an error, because he’s a dick, and now all your snowflakes are urine and you can’t even find the cat.

[http://www.stilldrinking.org/programming-sucks](url)

**Comment by marianadobre** [comment]
I am still a bit confused about that value 13. I just found an old email from Jim in which he mentioned that the last line should have 13 but that the value is stripped before it is sent to WEPP. He was talking about the old interface. So I don't understand why we had to have it there? I am certain the soils should not have 13 when they are in the runs folder so I am guessing that what we have now is fine. But we might still want to ask him this on Friday. Results are quite different with or without it.
